plugin/src/main/kotlin/net/octyl/ts2kt/gradle/repository/npm/NpmClientRepository.kt

```diff
@@ -62,8 +64,22 @@ class NpmClientRepository(private val project: Project) : ClientRepository {
     private val registryUrl: String by lazy {
         val cap = ByteArrayOutputStream()
 
+        val npm = with (OperatingSystem.current()) {
+            when {
+                // on Windows, NPM is a .bat file that needs to run under cmd.exe
+                isWindows -> arrayOf("cmd", "/c", "npm")
+                // TODO @kenzierocks: fill in whatever platform you build on, possibly Linux?
+                // isLinux -> arrayOf("npm")
+                else -> {
+                    // A good default, but this command may not work unless you have tested the platform. Warn the user.
```

Comment:
In what cases would this not work? Both Linux and macOS are *nix platforms, so the command will just be `npm` on them. I don't think Gradle supports any other non-*nix platforms besides Windows.

Reply:
Correct, but it's also an assumption. Unless you've tested on Mac.